src/reducers/sources.js

```diff
@@ -39,12 +39,11 @@ const State = makeRecord(({
   selectedLocation: undefined,
```

Comment:
perhaps we also want to store the `selectedLocation` or `pendingSelectedLocation ` in localStorage as well.